src/styles/blocks/page.scss

```diff
@@ -0,0 +1,7 @@
+.page {
```

Comment:
You placed both outer and inner properties in this selector. Not quite sure that it's allowed according to BEM. Better check it out.

Reply:
fixed, thanks)